### PR TITLE
remove ws hack

### PR DESCRIPTION
**Pull request by joske**

### CODE REVIEW

File: narwhal/benchmark/benchmark/logs.py [hunk 1]
```diff
@@ -349,13 +349,15 @@ def __init__(self, primaries, faults=0):
         # Parse the primaries logs.
         try:
             with Pool() as p:
+                time.sleep(1)
```
Comment (niklaslong) [inline narwhal/benchmark/benchmark/logs.py:352]: do we want to keep this in?

**Review by niklaslong — APPROVED** [review]
lgtm 👍

File: narwhal/benchmark/benchmark/logs.py (hunk 1 above)
Comment (joske) [reply narwhal/benchmark/benchmark/logs.py:352]: this is just a benchmark test

**Comment by joske** [comment]
remove ws hakari

**Pull request merged**